include/libdnf5-cli/output/smartcols_table_wrapper.hpp

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef LIBDNF5_CLI_OUTPUT_SMARTCOLS_TABLE_WRAPPER_HPP
+#define LIBDNF5_CLI_OUTPUT_SMARTCOLS_TABLE_WRAPPER_HPP
+
+#include <libsmartcols/libsmartcols.h>
+
+
+namespace libdnf5::cli::output {
+
+class SmartcolsTableWrapper {
+public:
+    SmartcolsTableWrapper();
+    ~SmartcolsTableWrapper();
```

Comment:
I think when assign/copy constructors are used, it will result in double de-allocation of memory or access of freed memory. The class contains raw pointers that are only copied as an address to memory. I am not sure whether move constructors also requires additional care.